device/accton/x86_64-accton_as7116_54x-r0/Accton-AS7116-54X/buffers.json.j2

```diff
@@ -0,0 +1,111 @@
+{# Default values which will be used if no actual configura available #}
+{% set default_cable = '40m' %}
+{% set default_speed = '10G' %}
+{% set default_ports_num = 54 -%}
+
+{# Port configuration to cable length look-up table #}
+{# Each record describes mapping of DUT (DUT port) role and neighbor role to cable length #}
+{# Roles described in the minigraph #}
+{% set ports2cable = {
+        'torrouter_server'       : '5m',
+        'leafrouter_torrouter'   : '40m',
+        'spinerouter_leafrouter' : '300m'
+        }
+%}
+
+{%- macro cable_length(port_name) -%}
```

Comment:
THis function looks familiar in buffer_config.j2. Can we leverage that template?

Reply:
Some customer don't want to binding the default buffer config ,so we leave it null let customer to config it by himself.